server/controllers/bookingControllers.js

```diff
@@ -1,10 +1,18 @@
 import moment from 'moment';
-import _ from 'lodash';
 import validate from '../middlewares/validateBooking';
-import users from '../models/usersModels';
-import trips from '../models/tripsModels';
 import response from '../helpers/response';
-import bookings from '../models/bookingsModels';
+//Database connection
+import {Client, Pool} from 'pg';
+import dotenv from 'dotenv';
+dotenv.config();
+const{JWT} = process.env;
+const {DATABASE_URL} = process.env;
+const connectionString = DATABASE_URL;
```

Comment:
I'm curious as to why you are reassigning the DATABASE_URL variable. Why not use it as it is?

Reply:
I have to hide it into .env file in order to protect the URL because it contains the database user and password